README.md

```diff
@@ -1,2 +1,38 @@
-# Portfolio-Mobile-Version-
-Setup a mobile version portfolio to display my projects and skills
+# Portfolio-Mobile-Version
+
+<h1>Setup a mobile version Portfolio: Mobile Version Projects PR to display my projects and skills</h1>
+
+<h2>Hi, I simple coded the mobile version with the help of flexbox and figma features.
+It is a structure where we decorate and furnishe further. Develop the responsive one,
+It's a Portfolio: Mobile Version projects display. It helps the users to understand 
+our services better. It also helps users to understand now a days features and techs. </h2>
+
+<h3>Built With Major languages Frameworks Technologies used Html, Css, linters.</h3>
+<h4>Grid,</h4>
+<h4>Flexbox,</h4>
+<h4>Hover</h4>
+
+<h5>To get a local copy up and running follow these simple example steps.</h5>
+
+Prerequisites 
+Setup 
+Install 
+Usage 
+Run 
+Tests
+Deployment 
+Authors 
+👤 Author
+
+GitHub: @ch-ahtesham-ul-haq Twitter: @chaury39 
+LinkedIn: https://www.linkedin.com/in/ch-ahtesham-ul-haq-b5318b152/
+
+🤝 Contributing Contributions, issues, and feature requests are welcome!
+
+Feel free to check the issues page.
+
+Show your support Give a ⭐️ if you like this project!
+
+Acknowledgments Hat tip to anyone whose code was used Inspiration etc 
+📝 <h6>License</h6>
+ This project is MIT licensed.
```

Comment:
- [ ] You did a great job, however, your Readme file still doesn't fit Microverse template. Kindly use this [template](https://github.com/microverseinc/readme-template/blob/master/README.md) as it  is. Just remove the sections that you don't need and add your personal informations. here is an [example.](https://github.com/algerina/My_portfolio/blob/main/README.md)

Reply:
Followed the template. Thank You